server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetIndicesAction.java

```diff
@@ -58,6 +71,9 @@ public String getName() {
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
         String[] indices = Strings.splitStringByCommaToArray(request.param("index"));
+        if (request.paramAsBoolean(INCLUDE_TYPE_NAME_PARAMETER, true)) {
```

Comment:
Another candidate for referring to BaseRestHandler.DEFAULT_INCLUDE_TYPE_NAME_POLICY but no big deal if you miss this